yarn.lock

```diff
@@ -4451,7 +4451,7 @@ mixin-deep@^1.2.0:
     for-in "^1.0.2"
     is-extendable "^1.0.1"
 
-mkdirp@^0.5.0, mkdirp@^0.5.1, mkdirp@^0.5.3:
+mkdirp@^0.5.0, mkdirp@^0.5.1:
```

Comment:
Wouldn't this still trigger the deprecation warning anyway?

Reply:
They are depended by our devDependencies or peerDependencies.
```
$ yarn why mkdirp

=> Found "mkdirp@0.5.3"
info Reasons this module exists
   - "@babel#cli" depends on it
   - Hoisted from "@babel#cli#mkdirp"
   - Hoisted from "eslint#mkdirp"
   - Hoisted from "webpack#mkdirp"
   - Hoisted from "nyc#spawn-wrap#mkdirp"
   - Hoisted from "@babel#cli#output-file-sync#mkdirp"
   - Hoisted from "ava#unique-temp-dir#mkdirp"
   - Hoisted from "webpack#terser-webpack-plugin#cacache#mkdirp"
   - Hoisted from "chokidar#fsevents#node-pre-gyp#mkdirp"
   - Hoisted from "chokidar#fsevents#node-pre-gyp#tar#mkdirp"
   - Hoisted from "webpack#terser-webpack-plugin#cacache#move-concurrently#mkdirp"
   - Hoisted from "eslint#file-entry-cache#flat-cache#write#mkdirp"
   - Hoisted from "webpack#terser-webpack-plugin#cacache#move-concurrently#copy-concurrently#mkdirp"
```